mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java

```diff
@@ -199,9 +199,15 @@ void copyReceiptLink() {
 
     @OnClick(R.id.transaction_reciept_share)
     void shareReceiptLink() {
+        String senderMsg =
```

Comment:
Also the variable name _senderMsg_ doesn't seem to make much sense(at least to me), as this was never written by the sender. As this is the content/body of the mail, so it should be named according to that in my opinion

Reply:
what about shareMsgBody, but I don't think now we need a variable because I'm going to create a constant for string now.